artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/jgroups/JChannelManager.java

```diff
@@ -34,7 +34,7 @@
 
    private static final Logger logger = Logger.getLogger(JChannelManager.class);
 
-   private Map<String, JChannelWrapper> channels;
```

Comment:
It is not needed anyways.

ChannelBroadcastEndpointFactory::singletonManager is already static.

Revert this please. If you want to make sure this is static, we could make the whole JChannelManager singleton. But I'm not sure now if there is a reason to not to.